cmd/createIdentity.go

```diff
@@ -0,0 +1,146 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/canonical/identity-platform-admin-ui/internal/authorization"
+	"github.com/canonical/identity-platform-admin-ui/internal/config"
+	"github.com/canonical/identity-platform-admin-ui/internal/kratos"
+	"github.com/canonical/identity-platform-admin-ui/internal/logging"
+	"github.com/canonical/identity-platform-admin-ui/internal/mail"
+	"github.com/canonical/identity-platform-admin-ui/internal/monitoring"
+	"github.com/canonical/identity-platform-admin-ui/internal/openfga"
+	"github.com/canonical/identity-platform-admin-ui/internal/pool"
+	"github.com/canonical/identity-platform-admin-ui/internal/tracing"
+	"github.com/canonical/identity-platform-admin-ui/pkg/identities"
+	"github.com/kelseyhightower/envconfig"
+	kClient "github.com/ory/kratos-client-go"
+	"github.com/spf13/cobra"
+)
+
+var createIdentityCmd = &cobra.Command{
+	Use:   "create-identity [jsonfile]",
+	Short: "Create an identity",
+	Example: `Create an example identity:
+
+cat > ./example.json <<EOF
+{
+	"schema_id": "default",
+	"traits": {
+		"email": "foo@example.com"
+	},
+	"credentials"": {
+		"password": {
+			"config": {
+				"password": "bar"
+			}
+		}
+	}
+}
+EOF
```

Comment:
thought: Would it make sense to simplify this by asking the user to inject it via command args? eg:
`admin-ui create user --password 123 --schema-id default --trait email=foo@example.com --trait name=Bob --trait phone=1234`

Reply:
yeah, however it would be a little difficult to represent nested traits I guess (correct me if I'm wrong). And the `trait` argument might go with a very long list.

We can update it whenever we find json input does not work conveniently. (changes will be limited to the argument parsing parting.)